src/obu.c

```diff
@@ -123,12 +127,13 @@ static uint32_t avifBitsReadVLC(avifBits * const bits)
 }
 
 // ---------------------------------------------------------------------------
-// Variables in here use snake_case to self-document from the AV1 spec:
+// Variables in here use snake_case to self-document from the AV1 and AV2 specs:
```

Comment:
this only describes AV1. You could say from the AV1 spec and the in development AV2 spec.